### DISCUSSION TITLE
Discord server

### DISCUSSION THREAD

**Discussion by senev3141** [zed-industries/zed#6556] [Feature Requests]
Hi, is there any discord server for Zed?

**Comment by iamnbutler** [comment]
Unsure why this doesn't show up in search but: https://github.com/zed-industries/community/discussions/1296#discussioncomment-5957683

**Comment by eastoncaleb** [comment]
The thread mentioned has been archived and all the invite links in it expired anyway. Can I get an invite to the discord server?

**Reply by Moshyfawn** [reply]
I'm not sure if the intention behind not including the Discord link on the official site is to slow down traffic or just an oversight, tbh.

Here's the link: https://discord.gg/dTy7kFJD

**Reply by eastoncaleb** [reply]
Thank you 👍🏻

**Comment by joelselvaraj** [comment]
Can I get an invite to the discord server too?

**Reply by Moshyfawn** [reply]
https://github.com/zed-industries/zed/discussions/6923#discussioncomment-8634654

**Reply by joelselvaraj** [reply]
The latest link in the thread worked. thanks!

**Reply by Moshyfawn** [reply]
Perfect, let's see how long it lasts 🤭

**Comment by balindam** [comment]
hey
the latest link has expired. Can anyone send an invite to the discord server?
